modules/cnf-du-pod-mutation.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-pod-mutation_{context}"]
+
+= Pod mutation
+
+The workload partitioning feature modifies pods that are annotated with `target.workload.openshift.io/{workload-type}` that are in a namespace that contains the `workload.openshift.io/allowed: {workload-type}` annotation.
+
+For example, a pod with the following CPU request specified would be mutated:
+----
```

Comment:
For the first two and the fourth, please mark the code blocks with [source,yaml].